### PR TITLE
Cleanup scala logs

### PR DESCRIPTION
**Pull request by mejdoo**
### Issues

In this PR , we (Mejdoo & Maciek) removed unnecessary Scala log statements.

#### APK
[Download build #936](http://10.10.124.11:8080/job/Pull%20Request%20Builder/936/artifact/build/artifact/wire-dev-PR2554-936.apk)
[Download build #937](http://10.10.124.11:8080/job/Pull%20Request%20Builder/937/artifact/build/artifact/wire-dev-PR2554-937.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/services/websocket/WebSocketService.scala [hunk 1]
```diff
@@ -32,53 +32,49 @@ import com.waz.service.{AccountsService, GlobalModule, ZMessaging}
...
+      accsInFG <- Signal.sequence(accs.map(_.selfUserId).map(id => accounts.accountState(id).map(st => id -> st)).toSeq: _*).map(_.toMap)
       (zmsWithWSActive, zmsWithWSInactive) =
-        accs.partition(zms => accsInFG(zms.selfUserId) == InForeground || wsForegroundEnabled || !cloudPushAvailable)
+      accs.partition(zms => accsInFG(zms.selfUserId) == InForeground || wsForegroundEnabled || !cloudPushAvailable)
     } yield (zmsWithWSActive, zmsWithWSInactive)
 
   lazy val serviceInForeground: Signal[Boolean] =
     for {
-      uiActive       <- global.lifecycle.uiActive
+      uiActive <- global.lifecycle.uiActive
```
Comment (BradleyWire) [inline app/src/main/scala/com/waz/services/websocket/WebSocketService.scala]: is this formatting correct?

File: app/src/main/scala/com/waz/zclient/appentry/AppEntryActivity.scala [hunk 2]
```diff
@@ -102,9 +102,9 @@ class AppEntryActivity extends BaseActivity {
     )
 
     Signal(accountsService.zmsInstances.map(_.nonEmpty), attachedFragment).map {
-      case (false, _)                                          => View.GONE
+      case (false, _) => View.GONE
       case (true, fragment) if fragmentTags.contains(fragment) => View.GONE
-      case _                                                   => View.VISIBLE
```
Comment (BradleyWire) [inline app/src/main/scala/com/waz/zclient/appentry/AppEntryActivity.scala]: I don't think this change abides by the current styling for scala

File: app/src/main/scala/com/waz/services/websocket/WebSocketService.scala (hunk 1 above)
Comment (mejdoo) [reply app/src/main/scala/com/waz/services/websocket/WebSocketService.scala]: fixed

File: app/src/main/scala/com/waz/zclient/appentry/AppEntryActivity.scala (hunk 2 above)
Comment (mejdoo) [reply app/src/main/scala/com/waz/zclient/appentry/AppEntryActivity.scala]: Fixed, auto formatting brakes the current Scala style.

Comment (BradleyWire) [reply app/src/main/scala/com/waz/zclient/appentry/AppEntryActivity.scala]: You can hi-light the changes you want to format and use auto-format on that. It won't change the whole class that way.

**Pull request merged**